build/Targets/Settings.targets

```diff
@@ -78,6 +78,7 @@
 
   <PropertyGroup Condition="'$(DevEnvDir)' == ''">
     <DevEnvDir>$(VSINSTALLDIR)\Common7\IDE</DevEnvDir>
+    <DevEnvDir Condition="'$(TF_BUILD_BUILDNUMBER)' != ''">$(VS140COMNTOOLS)\..\IDE</DevEnvDir>
```

Comment:
The point of my change was to stop hardcoding references to "14". What alternative options to do we have here?